Gopkg.toml

```diff
@@ -30,9 +30,5 @@
   unused-packages = true
 
 [[constraint]]
-  name = "github.com/spf13/cobra"
-  version = "0.0.3"
-
-[[constraint]]
-  name = "github.com/sensu/sensu-go"
```

Comment:
https://github.com/sensu/sensu-slack-handler/pull/19#discussion_r289935497

Reply:
Correct me if I am wrong. I think github.com/sensu/sensu-plugins-go-library library will pull in github.com/sensu/sensu-go as it has dependency on it.